### ISSUE TITLE
 youtube: Unable to extract Initial JS player n function name;

### ISSUE THREAD

**Issue by bennettpeter** [yt-dlp/yt-dlp#2602]
### Checklist

- [X] I'm reporting a bug unrelated to a specific site
- [X] I've verified that I'm running yt-dlp version **2022.01.21**. ([update instructions](https://github.com/yt-dlp/yt-dlp#update))
- [X] I've checked that all provided URLs are alive and playable in a browser
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/ytdl-org/youtube-dl#video-url-contains-an-ampersand-and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues including closed ones. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Description

Download is successful, however an error message is displayed with a request to report it here. See below log. I don't know if it is related to a specific site or a specific url.

### Verbose log

```shell
peter@office:~/Videos/recordings$ yt-dlp  -f 134 --verbose  "https://www.youtube.com/watch?v=rnpa9dTaJes"
[debug] Command-line config: ['-f', '134', '--verbose', 'https://www.youtube.com/watch?v=rnpa9dTaJes']
[debug] Encodings: locale UTF-8, fs utf-8, out utf-8, err utf-8, pref UTF-8
[debug] yt-dlp version 2022.01.21 [f20d607b0] (zip)
[debug] Python version 3.8.10 (CPython 64bit) - Linux-5.13.0-27-generic-x86_64-with-glibc2.29
[debug] exe versions: ffmpeg 4.2.4, ffprobe 4.2.4
[debug] Optional libraries: Crypto, secretstorage, sqlite
[debug] Proxy map: {}
[debug] [youtube] Extracting URL: https://www.youtube.com/watch?v=rnpa9dTaJes
[youtube] rnpa9dTaJes: Downloading webpage
[youtube] rnpa9dTaJes: Downloading android player API JSON
[youtube] rnpa9dTaJes: Downloading player e06dea74
WARNING: [youtube] nsig extraction failed: You may experience throttling for some formats
n = 3aPOeBIOrDRrUQdtNA ; player = https://www.youtube.com/s/player/e06dea74/player_ias.vflset/en_US/base.js
rnpa9dTaJes: Traceback (most recent call last):
  File "/home/peter/bin/yt-dlp/yt_dlp/extractor/youtube.py", line 2412, in _decrypt_nsig
    self._player_cache[player_id] = self._extract_n_function(video_id, player_url)
  File "/home/peter/bin/yt-dlp/yt_dlp/extractor/youtube.py", line 2433, in _extract_n_function
    funcname = self._extract_n_function_name(jscode)
  File "/home/peter/bin/yt-dlp/yt_dlp/extractor/youtube.py", line 2421, in _extract_n_function_name
    return self._search_regex(
  File "/home/peter/bin/yt-dlp/yt_dlp/extractor/common.py", line 1186, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
yt_dlp.utils.RegexNotFoundError: Unable to extract Initial JS player n function name; please report this issue on  https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; type  yt-dlp -U  to update. Be sure to call yt-dlp with the --verbose flag and include its complete output.
 (caused by RegexNotFoundError('Unable to extract \x1b[0;94mInitial JS player n function name\x1b[0m; please report this issue on  https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; type  yt-dlp -U  to update. Be sure to call yt-dlp with the --verbose flag and include its complete output.')); please report this issue on  https://github.com/yt-dlp/yt-dlp . Make sure you are using the latest version; type  yt-dlp -U  to update. Be sure to call yt-dlp with the --verbose flag and include its complete output.
[debug] Sort order given by extractor: quality, res, fps, hdr:12, source, codec:vp9.2, lang, proto
[debug] Formats sorted by: hasvid, ie_pref, quality, res, fps, hdr:12(7), source, vcodec:vp9.2(10), acodec, lang, proto, filesize, fs_approx, tbr, vbr, abr, asr, vext, aext, hasaud, id
[info] rnpa9dTaJes: Downloading 1 format(s): 134
[debug] Invoking downloader on "https://rr1---sn-jvhj5nu-cvne7.googlevideo.com/videoplayback?expire=1643837742&ei=zqT6YfvVLoL68wSlxqyoCw&ip=2601%3A183%3A100%3A3220%3A690f%3Ac175%3Afbea%3A49e2&id=o-AI8YFqHHwbw1MH-RM0vv4XrhUkA8VLe2sXtyNFHf46Fb&itag=134&source=youtube&requiressl=yes&mh=-U&mm=31%2C29&mn=sn-jvhj5nu-cvne7%2Csn-ab5l6n6s&ms=au%2Crdu&mv=m&mvi=1&pl=34&initcwndbps=1553750&vprv=1&mime=video%2Fmp4&gir=yes&clen=5466180&dur=277.977&lmt=1629216240517080&mt=1643815982&fvip=1&keepalive=yes&fexp=24001373%2C24007246&c=ANDROID&txp=5316224&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cvprv%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRgIhAPa4uBl6Jx8P5Z9ikgT2kE9o9dDi41jbk1qjP1D9dMLgAiEAvC8intCUXMh_54Ckx2jpszW7W2B1SvBbWIqXyGX3gHM%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRAIgLyAtSLi-TgQYd74atUxjO5v7E5g61Iqg2IXp88skcqECIHvKv-VdTn6l3QtAKhmzbJwxVZO_MiYiM2mD-2aTtd0X"
[download] How to Replace Acer Nitro 5 Keys [rnpa9dTaJes].mp4 has already been downloaded
[download] 100% of 5.21MiB
```

**Comment by user7230724** [comment]
Same error here.

**Comment by Lesmiscore** [comment]
Duplicate of #2576, #2569 (and some more)

Fixed in 48416bc4a, will be in next release. Guys, please make sure to search issues first.
